README.md

```diff
@@ -1,6 +1,121 @@
-# sensu-puppet-handler
+# Sensu Puppet Keepalive Handler
 
-Deregister Sensu entities if they no longer have an associated Puppet node. The
-puppet handler requires access to a SSL truststore and keystore, containing a
-valid (and whitelisted) Puppet certificate, private key, and CA. The local
-Puppet agent certificate, private key, and CA can be used.
\ No newline at end of file
+- [Overview](#overview)
+- [Usage examples](#usage-examples)
+- [Configuration](#configuration)
+  - [Asset registration](#asset-registration)
+  - [Handler definition](#handler-definition)
+  - [Check definition](#check-definition)
+- [Installation from source and
+  contributing](#installation-from-source-and-contributing)
+
+## Overview
+
+The [Sensu Puppet Keepalive Handler][0] is a [Sensu Event Handler][3] that will
+delete an entity with a failing keepalive check when its corresponding [Puppet][2]
+node no longer exists.
+
+## Usage examples
+
+Help:
+
+```
+Usage:
+  sensu-puppet-handler [flags]
+  sensu-puppet-handler [command]
+
+Available Commands:
+  help        Help about any command
+  version     Print the version number of this plugin
+
+Flags:
+      --cacert string              path to the site's Puppet CA certificate PEM file (default "ca.pem")
```

Comment:
I might suggest the following for consistent styling (ex. https://github.com/sensu/sensu-go/blob/572b4b5facc4a4f7b310dcf0eb34ef711e9583e5/backend/cmd/start.go#L40)
```suggestion
      --ca-cert string              path to the site's Puppet CA certificate PEM file (default "ca.pem")
```